samples/exportAssets.js

```diff
@@ -50,7 +50,38 @@ async function main(dumpFilePath) {
     // Do things with with the response.
     console.log(result);
   }
+
+  async function exportAssetsBigquery() {
```

Comment:
If possible, let's pull this into its own sample file `samples/exportAssetsBigquery.js`.

Reply:
Done